lib/challenge.js

```diff
@@ -51,7 +67,17 @@ var pet_owner;
 var some_pet;
 
 // Answer Starts Here
-
+var some_pet = {
+  name: "Rupert"
+  species: "Dog"
+  breed: "Goldendoodle"
+  noise: "Bark"
+  makeNoise: function makeNoise = {
+    if (species === "Dog") {
+      return "Bark"
+    }
```

Comment:
Mmm... not quite what we were looking for. What if `some_pet.noise` isn't "bark" but, rather, "woof"? `makeNoise` should reference the `noise` property of `some_pet`. In other words,
```javascript
makeNoise : function() {
  return this.noise;
}
```